windows/src/engine/keyman32/globals.h

```diff
@@ -151,7 +151,7 @@ class Globals
 /* External interface functions */
 
 typedef HRESULT (WINAPI *PKEYMANPROCESSOUTPUTFUNC)(int n, WCHAR *buf, int nbuf);
-typedef HRESULT (WINAPI *PKEYMANGETCONTEXTFUNC)(int n, PWSTR buf);
+typedef HRESULT (WINAPI *PKEYMANGETCONTEXTFUNC)(int n, PWSTR buf, BOOL* isTextSelected);
```

Comment:
One concern here is that changing an existing API endpoint can be catastrophic if we end up with divergent versions of the DLLs loaded (e.g. if a user doesn't restart Windows after upgrading Keyman -- then any processes with Keyman loaded can crash). Is is usually safer to add another API and mark the original as deprecated; the same checks for whether the API is available can be used.